src/models/environment-variable/updatable-environment-variable.js

```diff
@@ -0,0 +1,6 @@
+import PropTypes from 'prop-types';
+
+export default Object.freeze({
```

Comment:
Only imported by use-save-env-var.js, but the import is not used there. Can probably be deleted.

Reply:
It is used in property definition
```
import updatableEnvVar from '../../models/environment-variable/updatable-environment-variable';
...
useSaveEnvVar.propTypes = {
...
  updatableEnvVars: PropTypes.arrayOf(PropTypes.shape(updatableEnvVar))
    .isRequired,
```